Source/Diagnostics/WarpXIO.cpp

```diff
@@ -334,6 +334,8 @@ WarpX::InitFromCheckpoint ()
         }
     }
 
+    InitializeEBGridData(maxLevel());
+
```

Comment:
Does the EB only exist on the finest level? If not, should this be in the loop over levels just above?

Reply:
It only exists on the finest level right now - in `InitializeEBGridData`, the first thing it does is return if `lev != maxLevel()`.